src/openms/include/OpenMS/ANALYSIS/OPENSWATH/PeakIntegrator.h

```diff
@@ -77,7 +168,10 @@ namespace OpenMS
       @param[in] left The left retention time boundary
       @param[in] right The right retention time boundary
     */
-    void estimateBackground(const MSChromatogram& chromatogram, const double& left, const double& right);
+    void estimateBackground(
+      const MSChromatogram& chromatogram, const double& left, const double& right, const double& peak_apex_pos,
+      double& background_area, double& background_height
```

Comment:
note how this will not work with pyOpenMS as Python does not support passing primitive data types (double, int) by reference. You will not be able to get these data back.